src/Microsoft.Data.SqlClient/netfx/src/BID/INC/CS/BidPrivateBase.cs

```diff
@@ -131,1139 +55,58 @@ internal static bool IsOn(ApiGroup flag)
-        internal static void DllBidEntryPoint(ref IntPtr hID, int bInitAndVer, string sIdentity,
-                                            uint propBits, ref ApiGroup pGblFlags, CtrlCB fAddr,
-                                            ref BIDEXTINFO pExtInfo, IntPtr pHooks, IntPtr pHdr);
-
-        [ResourceExposure(ResourceScope.Machine)]
-        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.Success)]
-        [DllImport(dllName)]
-        extern
-        internal static void DllBidEntryPoint(ref IntPtr hID, int bInitAndVer, IntPtr unused1,
-                                            uint propBits, ref ApiGroup pGblFlags, IntPtr unused2,
-                                            IntPtr unused3, IntPtr unused4, IntPtr unused5);
-
-        [ResourceExposure(ResourceScope.None)]
-        [DllImport(dllName)]
-        extern
-        internal static void DllBidInitialize();
-
-        [ResourceExposure(ResourceScope.None)]
-        [ReliabilityContract(Consistency.WillNotCorruptState, Cer.Success)]
-        [DllImport(dllName)]
-        extern
-        internal static void DllBidFinalize();
-
-    } // NativeMethods
+    private static ApiGroup modFlags = ApiGroup.Off;
 
 } // Bid{PrivateBase}
 
-//+//////////////////////////////////////////////////////////////////////////////////////////////
-//                                                                                             //
-//                                         Attributes                                          //
-//                                                                                             //
-//-//////////////////////////////////////////////////////////////////////////////////////////////
-//
-//  [module: BidIdentity("ModuleIdentityString")]
-//
-[AttributeUsage(AttributeTargets.Module, AllowMultiple = false)]
-internal sealed class BidIdentityAttribute : Attribute
-{
-    internal BidIdentityAttribute(string idStr)
-    {
-        _identity = idStr;
-    }
-    internal string IdentityString
-    {
-        get { return _identity; }
-    }
-    string _identity;
-}
-
-//
-//  [module: BidMetaText("<Alias> ...")]
-//  [module: BidMetaText("<ApiGroup> ...")]
-//  ...etc...
-//
-[AttributeUsage(AttributeTargets.Module, AllowMultiple = true)]
-internal sealed class BidMetaTextAttribute : Attribute
-{
-    internal BidMetaTextAttribute(string str)
-    {
-        _metaText = str;
-    }
-    internal string MetaText
-    {
-        get { return _metaText; }
-    }
-    string _metaText;
-}
-
 
-/// <summary>
-/// This attribute is used by FxCopBid rule to mark methods that accept format string and list of arguments that match it
-/// FxCopBid rule uses this attribute to check if the method needs to be included in checks and to read type mappings
-/// between the argument type to printf Type spec.
-/// 
-/// If you need to rename/remove the attribute or change its properties, make sure to update the FxCopBid rule!
-/// </summary>
```

Comment:
Don't remove the summary so we know not to remove or rename this in the future.

Reply:
Added the summary and attributes back.